init.lua

```diff
@@ -1,6 +1,15 @@
 digilines = {}
 digilines.S = minetest.get_translator("digilines")
 
+digilines.mtg = minetest.get_modpath("default")
+digilines.mcl = minetest.get_modpath("mcl_core")
+
+-- sounds check
+digilines.sounds = {}
```

Comment:
This default value is not better than `nil` because `digilines.sounds.node_sound_wood_defaults()` is a function call.

Reply:
removed then

Reply:
Maybe another game without `default` or without `mcl_sounds` wants to use this mod, I think we should not force the error, I think setting `digilines.sounds = {}` avoids this error and allows more compatibility